components/cam/src/physics/cam/shoc.F90

```diff
@@ -46,6 +46,8 @@ module shoc
 real(r8), parameter :: w2tune=1.0_r8 
 ! third moment of vertical velocity
 real(r8), parameter :: w3clip=1.2_r8 
+! mixing length scaling parameter
+real(r8), parameter :: length_fac=1.0_r8
```

Comment:
Should these be namelist parameters, or do we expect to hold them constant in general?

Reply:
no, i do not advocate these being in the namelist.  With that said, there are other SHOC parameters I think should be in the namelist (still in the process of determining exactly which), but that is the subject for a separate PR.